### PR TITLE
Add description and embedded css to POS

### PR DESCRIPTION
**Pull request by Kukks**

### CODE REVIEW

File: BTCPayServer/Views/AppsPublic/ViewPointOfSale.cshtml [hunk 1]
```diff
@@ -42,11 +42,16 @@
...
-                max-width: 320px;
-                margin: auto;
-            }
+        .card-deck .card:only-of-type {
+            max-width: 320px;
+            margin: auto;
+        }
+        @if (!string.IsNullOrEmpty(Model.EmbeddedCSS))
+        {
+                @Safe.Raw(Model.EmbeddedCSS);
```
Comment (NicolasDorier) [inline BTCPayServer/Views/AppsPublic/ViewPointOfSale.cshtml]: will not work, `Raw` can only be used in the context of an HTML doc, but you are inside the style tag. Look what I did when I fixed the vulnerability.

Comment (Kukks) [reply BTCPayServer/Views/AppsPublic/ViewPointOfSale.cshtml]: hmm but it did work for me?

Comment (NicolasDorier) [reply BTCPayServer/Views/AppsPublic/ViewPointOfSale.cshtml]: `@Safe.Raw` is meant to insert HTML. Not using it in this way mean there is probably a corner case that is not handled correctly, I am not comfortable with that. Please, just add the `<style></style>` tags to the `EmbeddedCSS` directly.

**Comment by Kukks** [comment]
done
ᐧ

**Pull request merged**